libevm/VM.cpp

```diff
@@ -214,19 +214,25 @@ void VM::interpretCases()
 		
 		CASE(CREATE)
 		{
+			if (m_schedule->haveStaticCall && m_ext->staticCall)
+				throwBadInstruction();
+
 			m_bounce = &VM::caseCreate;
 		}
 		BREAK
 
 		CASE(DELEGATECALL)
-
-			// Pre-homestead
-			if (!m_schedule->haveDelegateCall)
-				throwBadInstruction();
-
+		CASE(STATICCALL)
 		CASE(CALL)
 		CASE(CALLCODE)
 		{
+			// Pre-homestead
```

Comment:
Remove this comment. It does not provided any information any more, as we are so used to hard-forks.